libraries/AP_Winch/AP_Winch.cpp

```diff
@@ -101,6 +102,56 @@ void AP_Winch::set_desired_rate(float rate)
     config.control_mode = ControlMode::RATE;
 }
 
+//Implement DO_WINCH Deliver message
+void AP_Winch::package_update()
+{
+    if ( deliveryFlag == true)
+    {
+        config.rate_desired = get_rate_max(); //negative is up, positive is down
+        config.control_mode = ControlMode::RATE;
+        GCS_SEND_TEXT(MAV_SEVERITY_INFO,"Package Requested, package lowering");
+        if ( touchGroundFlag == true )
+        {
+            GCS_SEND_TEXT(MAV_SEVERITY_INFO,"TOUCHED GROUND");
+            config.rate_desired = 0;
+            config.control_mode = ControlMode::RATE;
+            deliveryFlag = { false };
+            reelUpFlag = { true };
+        }
+    }
+
+    //could be moved into own function later. For Daiwa, 540 is empty line weight, 240 is max line weight. current threshold at 420
```

Comment:
540 for empty and 240 for max weight? Better to convert to some standard unit I would have expected the empty number to be less than the max weight one.